src/app/(tabs)/story/index.tsx

```diff
@@ -13,17 +14,35 @@ import { RenderHTML } from 'react-native-render-html';
 import { SafeAreaView } from 'react-native-safe-area-context';
 
 import styles from './styles';
-import { storyObject } from '../../../utils/story';
+import { fetchStory } from '../../../queries/stories';
+import { Story } from '../../../queries/types';
 
 function StoryScreen() {
   const [isLoading, setLoading] = useState(true);
   const scrollRef = React.useRef<any>(null);
-  const [story, setStory] = useState<any>();
+  const [story, setStory] = useState<Story>();
```

Comment:
one workaround to all the null checks is changing the initial value of the story to be like this: useState<Story>({} as Story). You can then remove the ? and ! that are being used without warnings I believe